conda_build.recipe/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: numpy
+  version: 1.11.0{{ '+' + environ.get('GIT_DESCRIBE_NUMBER', '0') + '.' + environ.get('GIT_DESCRIBE_HASH', '0') if environ.get('GIT_DESCRIBE_NUMBER', '0') != '0' else "" }}
```

Comment:
No worries. There are other ways to get at the build string. For instance, we can set the version at build time by extracting it from `python setup.py --version`. We will leave some filler version here, but it won't be an issue if it isn't correct. I'll update what I am doing here to follow that strategy.